### DISCUSSION TITLE
Is there versioning for extensions?

### DISCUSSION THREAD

**Discussion by Jeinhaus** [datenstrom/community#622] [Ask a question] [answered]
I'm trying to set up a local environment that matches the server's environment and I would like to declaratively define the plugin versions used.

Any reason why you don't use tags in the [yellow-extension](https://github.com/datenstrom/yellow-extensions) repository? This would allow referencing the correct download url in the `update-current.ini`. It would also allow upgrading to specific versions using the CLI.

**Comment by markseuffert** [comment, marked as answer]
No, there’s no semantic versioning or tagging of certain points in history. Perhaps after version 1.0.
